Cargo.lock

```diff
@@ -391,6 +391,20 @@ dependencies = [
  "servo-freetype-sys 4.0.3 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "fuchsia-zircon"
```

Comment:
WR is going to ship parts of Fuchsia?

Reply:
Looks like it's a new dependency of mio?

Reply:
From Mio Cargo.toml

```toml
[target.'cfg(target_os = "fuchsia")'.dependencies]
fuchsia-zircon = "0.3.2"
fuchsia-zircon-sys = "0.3.2"
```

Reply:
It has `BSD-3-Clause` license, which is atypical for Rust ecosystem. Gotta check with Gecko folks before merging, since all dependencies are vendored for Gecko, and licenses matter.

Reply:
These `fuchsia-zircon` crates are already in the Gecko source tree because the latest version of `rand` depends on them too.